actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -192,11 +192,10 @@ def https?
       # HTTP methods in integration tests. +#process+ is only required when using a
       # request method that doesn't have a method defined in the integration tests.
       #
-      # This method returns a Response object, which one can use to
-      # inspect the details of the response. Furthermore, if this method was
+      # This method returns a response status. Furthermore, if this method was
       # called from an ActionDispatch::IntegrationTest object, then that
-      # object's <tt>@response</tt> instance variable will point to the same
-      # response object.
+      # object's <tt>@response</tt> instance variable will point to response object
```

Comment:
Nice catch, @vipulnsward. I didn't saw that this typo was present. Updated with PR.